examples/benchmarks/bubblebench/editable-shared-tree/package.json

```diff
@@ -39,6 +36,8 @@
 		"test:jest": "jest",
 		"test:jest:verbose": "cross-env FLUID_TEST_VERBOSE=1 jest",
 		"tsc": "tsc",
+		"tsfmt": "tsfmt --verify",
```

Comment:
these lines were removed in https://github.com/microsoft/FluidFramework/commit/2e295e0a6be407f773cf4b3b756f13127dd00c34#diff-3c675c6b50addad660dbb73bc376ae3b0b39383a2a9c54ca101d8806b196644d . This loses that cleanup.